src/worker/state.rs

```diff
@@ -251,14 +251,13 @@ impl WorkerState {
         }
     }
 
-    pub fn remove_task(&mut self, task_ref: TaskRef, just_finished: bool) {
-        let mut task = task_ref.get_mut();
+    fn remove_task(&mut self, task: &mut Task, task_ref: &TaskRef, just_finished: bool) {
```

Comment:
The `just_finished` attribute now only serves for asserts?

Reply:
yes